src/notation/internal/notationinteraction.cpp

```diff
@@ -2163,15 +2163,14 @@ bool NotationInteraction::isGripEditStarted() const
     return m_gripEditData.element && m_gripEditData.curGrip != Ms::Grip::NO_GRIP;
 }
 
-static int findGrip(const QVector<mu::RectF>& grips, Ms::Page* page, const mu::PointF& canvasPos)
+static int findGrip(const QVector<mu::RectF>& grips, const mu::PointF& canvasPos)
 {
-    if (grips.empty() || page == nullptr) {
+    if (grips.empty()) {
         return -1;
     }
-    mu::PointF pos = canvasPos - page->pos();
```

Comment:
odd, because I could swear until I added this, it wasn't working previously either...but it's definitely wrong